django_forms_encoder/encoder.py

```diff
@@ -0,0 +1,56 @@
+"""Django forms encoder."""
+from enum import Enum, unique
+from typing import Any, Dict
+
+from django.core.serializers.json import DjangoJSONEncoder
+from django.forms import Form, widgets
+
+
+@unique
+class WidgetType(str, Enum):
+    """Widget type enum."""
+
+    INPUT = 'input'
+    TEXTAREA = 'textarea'
+
+
+class DjangoFormsEncoder(DjangoJSONEncoder):
+    """Django forms encoder."""
+
+    def default(self, obj):
+        """Encode django form."""
+        if isinstance(obj, Form):
+            form: Dict[str, Any] = {
+                'fields': [],
+            }
+            if obj.non_field_errors():
+                form['form_errors'] = obj.non_field_errors()
+
+            for name, field in obj.fields.items():
+                field_obj = {
+                    'name': name,
+                    'label': field.label or name,
+                    'required': field.required,
+                }
+
+                if field.help_text:
+                    field_obj['help_text'] = field.help_text
+
+                if obj.errors.get(name):
+                    field_obj['field_errors'] = obj.errors.get(name)
```

Comment:
With newer version of schema there should be just `errors`

Reply:
Yeah, I'll update the code as soon as the schema is final.

Reply:
I pushed changes reflecting the new interface version.